programs/demo_common.sh

```diff
@@ -0,0 +1,137 @@
+## Common shell functions used by demo scripts programs/*/*.sh.
```

Comment:
Was it intentional to pull this file in? Looks like an accident.

Reply:
Och no. Sorry for that. Will remove redundant files.

Reply:
Should be fine now.